include/pros/rtos.hpp

```diff
@@ -669,4 +669,4 @@ using pros::c::delay;
 }  // namespace rtos
 }  // namespace pros
 
-#endif  // _PROS_RTOS_HPP_
+#endif  // _PROS_RTOS_HPP_
```

Comment:
add back newline please
```suggestion
#endif  // _PROS_RTOS_HPP_

```